.bazelrc

```diff
@@ -109,10 +109,20 @@ build --test_env=ASAN_SYMBOLIZER_PATH
 # `en_US`.
 build --action_env=LANG=en_US.UTF-8
 
-# Enable libpfm for google_benchmark on Linux only.
+# Allow per-platform configuration.
 build --enable_platform_specific_config
+
+# Enable libpfm for google_benchmark on Linux only.
 build:linux --define=pfm=1
 
+# Enable split debug info on Linux, which is significantly more space efficient
+# and should work well with modern debuggers. Note that this is not currently
+# supported on macOS.
```

Comment:
FWIW the MacOS debug info model looks a lot like Split DWARF already - I wouldn't expect they'll ever support actual Split DWARF because they've got a thing that works for them.
(The MacOS model moves more debug info out of the linked executable (including the line table, for instance) - but leaves it in the objects (there's a way to use Split DWARF that does this too - so you only have .o files, but the debug info in them still doesn't get linked in) - this has the build artifact size benefits, but does still mean you have to have all those debug info bytes available to the linker, even if it ignores them - not as good for distributed builds)

Reply:
Tweaked the comment. The fact that debug info remains in object files is still unfortunate because it means it gets copied into `.a` files from what I can see even though it doesn't make it all the way into the executable. Eventually we can maybe skip building `.a`s when we can use LLD on macOS.

Anyways, unlikely to change so made it more clear that it's just a Linux-specific option.

Reply:
Oh, yeah - .a files. Yep, avoiding those would help on MacOS, and still help a bit on Linux too - fewer unique bytes on rebuild, etc.